diff --git a/.gitmodules b/.gitmodules
index 984113151de..09ef878e8d5 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -36,8 +36,8 @@
 	url = https://github.com/rust-lang/edition-guide.git
 [submodule "src/llvm-project"]
 	path = src/llvm-project
-	url = https://github.com/rust-lang/llvm-project.git
-	branch = rustc/11.0-2020-10-12
+	url = https://github.com/espressif/llvm-project
+	branch = xtensa_release_11.0.0
 [submodule "src/doc/embedded-book"]
 	path = src/doc/embedded-book
 	url = https://github.com/rust-embedded/book.git
